### PR TITLE
Put bad glyphs in config

### PR DESCRIPTION
**Pull request by ecmendenhall**
- Add a distinct `*n-OU` locality for both Rails and JS. (This can possibly work for other langs too!)
- Undo automatic trial data adaptations.
- Load invalid symbol rgx via `Oulipo.fifth_glyph_r*g*x`, not constant
- Add `Oulipo::Config` to swap invalid symbols according to config

screen shot 2017-04-25 at 18 04 32

screen shot 2017-04-25 at 18 04 49

### CODE REVIEW

**Comment by ecmendenhall** [comment]
Hi @mouse-reeve! I think this approach to automatic trial data is a plus for maintainability. This will undo trial data modifications in favor of using config to switch lang and constraints. For all main Mastodon trials, it will run with lang `en` and no constraints, but for Oulipo trials, it will run with lang `en-OU` and our constraints. It won't be as painful to pull in stuff from Mastodon's main branch. It's also a foothold for additional Oulipo translations—`de-OU`, `fr-OU`, &c. i18n config could possibly hold "invalid glyph" configuration lang-by-lang, too! 

(This is a branch-off-a-branch, so DNM right now! Just want to ask your opinion on this approach!)

**Comment by mouse-reeve** [comment]
I think this is a good plan, both for conflicts and for maintainability

File: app/helpers/settings_helper.rb [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 module SettingsHelper
   HUMAN_LOCALES = {
-    en: 'Oulipo',
+    'en-OU': 'English (Oulipo)',
```
Comment (ecmendenhall) [inline app/helpers/settings_helper.rb:5]: Want to point this out: this brings back that glyph on localization options. Swap it if you want!

File: config/locales/en-OU.yml [hunk 2]
```diff
@@ -0,0 +1,312 @@
+---
+en-OU:
+  activerecord:
+    errors:
+      messages:
+        record_invalid: "Invalid: %{errors}"
```
Comment (ecmendenhall) [inline config/locales/en-OU.yml]: This bit will fix #38.

File: config/locales/en-OU.yml [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+      default: "%b %d, %Y, %H:%M"
+  two_factor_auth:
+    code_hint: Submit digits that your auth app displays to confirm
+    description_html: If you allow <strong>two-factor login</strong>, a string of digits will go to you via sms or an auth app, which you will add to your login form.
+    disable: Disallow
+    enable: Allow
+    enabled_success: Your two-factor login took!
+    generate_recovery_codes: Build backup strings
+    instructions_html: "<strong>Scan this QR visual into an app</strong>. From now on, that app will know your string of digits for login."
+    lost_recovery_codes: Recovery codes allow you to regain access to your account if you lose your phone. If you've lost your recovery codes, you can regenerate them here. Your old recovery codes will be invalidated.
```
Comment (ecmendenhall) [inline config/locales/en-OU.yml:304]: Also, a handful of strings down in this bit employ that bad glyph.

**Comment by ecmendenhall** [comment]
OK, @mouse-reeve, I think this is all good to go! It will also fix #38. I put a handful of thoughts down in this diff for you. Look at this in particular: https://github.com/mouse-reeve/mastodon/pull/34/files#r113320587

**Pull request merged**